extensions/interactions/Continue/directives/continue-validation.service.spec.ts

```diff
@@ -111,6 +111,10 @@ describe('ContinueValidationService', () => {
   it('should expect a non-confusing and non-null default outcome',
     () => {
       var warnings = validatorService.getAllWarnings(
```

Comment:
here we expect a `non-null` default outcome and hence the type of  parameter `outcome` is not `Outcome | null`